Composite/Core/WebClient/Renderings/RenderingContext.cs

```diff
@@ -243,7 +244,7 @@ private void InitializeFromHttpContextInternal()
 
             _dataScope = new DataScope(Page.DataSourceId.PublicationScope, culture);
             Thread.CurrentThread.CurrentCulture = culture;
-            Thread.CurrentThread.CurrentUICulture = culture;
+            Thread.CurrentThread.CurrentUICulture = HttpContext.Current?.Items["C1_CurrentCulture"] as CultureInfo ?? culture;

```

Comment:
Does `C1_CurrentCulture` get set anywhere? Can't recall having seen it before and a quick search turns up empty https://github.com/Orckestra/C1-CMS-Foundation/search?q=C1_CurrentCulture&unscoped_q=C1_CurrentCulture